testing/btest/Baseline/scripts.base.frameworks.input.sqlite.types/out

```diff
@@ -12,5 +12,5 @@ BB
 
 }, vc=[10, 20, 30], vs=[], vn=<uninitialized>]
 0
-1
+0
```

Comment:
This is incorrect now. In the input data, `p$vs` is a vector of string that contains one zero-length element. The vector thus has length 1.

(Sorry - I did not check what causes this change, I was just curious about the baseline change).

Some of the changes in setspecialcases also look suspicious - e.g. where s=[, , , , ,] is replaced with an empty set. I did not verify that - but I assume it is the same problem.

Note that you probably have to handle the case of a string set like "a," differently in the config framework than in the general ascii reader. In the config reader you probably just want to handle that like a trailing comma. In the ascii reader you probably want to take the strict interpretation, which is "two string elements, one being an "a", one being an empty string".

Reply:
Ok, completely reworked this to just change the regex in the config framework (which I swear I tried earlier). The rest of the changes were reverted.

Reply:
One question - looking at the PR, does this now accept something like

````
a,b,c,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

as meaning `a,b,c`?

Reply:
That's correct.

Reply:
Ok, next question - do we really want to allow that? I mean - I get one comma at the end, but that seems a bit excessive.
Though, to be fair, I am not sure that I really see a lot of drawbacks...

Reply:
Fair - thinking about it, I think I am fine with both alternatives. I am honestly not sure which I prefer anymore.